src/tools/perf/perftest_context.h

```diff
@@ -0,0 +1,63 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2024. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCX_PERFTEST_CONTEXT_H
+#define UCX_PERFTEST_CONTEXT_H
+
+#include "api/libperf.h"
+#include "lib/libperf_int.h"
+
+#define MAX_BATCH_FILES         32
+#define MAX_CPUS                1024
+
```

Comment:
1. Pls use 2 space lines in H file
2. Why needed to move them to a separate file?